examples/hybrid/advection.jl

```diff
@@ -0,0 +1,99 @@
+push!(LOAD_PATH, joinpath(@__DIR__, "..", ".."))
+
+using Test
+using StaticArrays, IntervalSets, LinearAlgebra
+
+import ClimaCore:
+    ClimaCore,
+    slab,
+    Spaces,
+    Domains,
+    Meshes,
+    Geometry,
+    Topologies,
+    Spaces,
+    Fields,
+    Operators
+import ClimaCore.Domains.Geometry: Cartesian2DPoint
+
+# set up function space
+function hvspace_2D()
+    FT = Float64
+    vertdomain =
+        Domains.IntervalDomain(FT(-1), FT(1); x3boundary = (:bottom, :top))
+    vertmesh = Meshes.IntervalMesh(vertdomain, nelems = 64)
+    vert_center_space = Spaces.CenterFiniteDifferenceSpace(vertmesh)
+    vert_face_space = Spaces.FaceFiniteDifferenceSpace(vert_center_space)
+
+    horzdomain = Domains.RectangleDomain(
+        -1..1,
+        -0..0,
+        x1periodic = true,
+        x2boundary = (:a, :b),
+    )
+    horzmesh = Meshes.EquispacedRectangleMesh(horzdomain, 10, 1)
+    horztopology = Topologies.GridTopology(horzmesh)
+
+    quad = Spaces.Quadratures.GLL{8}()
```

Comment:
General question. Is this setting the polynomial degree, the number or finite element nodes, or the number of quadrature points?

Reply:
@simonbyrne, this is for you.

Reply:
8 Refers to the number of quadrature points

Reply:
Thanks. And where is the polynomial degree set?